src/ci/docker/dist-armv7-linux/crosstool-ng.sh

```diff
@@ -0,0 +1,11 @@
+set -ex
+
+url="https://github.com/crosstool-ng/crosstool-ng/archive/crosstool-ng-1.24.0.tar.gz"
```

Comment:
We should mirror this to `rust-lang-ci-mirrors`.

Reply:
Good point, I've uploaded this to https://rust-lang-ci-mirrors.s3-us-west-1.amazonaws.com/rustc/crosstool-ng-1.24.0.tar.gz

@msizanoen1 if you get a chance, mind updating the URL to download from here and leaving a comment saying that the current URL is where it was originally sourced from?